applications/FluidDynamicsApplication/custom_elements/embedded_fluid_element.cpp

```diff
@@ -199,12 +199,71 @@ void EmbeddedFluidElement<TBaseElement>::Calculate(
     TBaseElement::Calculate(rVariable, rOutput, rCurrentProcessInfo);
 }
 
+///////////////////////////////////////////////////////////////////////////////////////////////////
+// Access
+
+template <class TBaseElement>
+void EmbeddedFluidElement<TBaseElement>::GetValueOnIntegrationPoints(
+    const Variable<array_1d<double, 3>> &rVariable,
+    std::vector<array_1d<double, 3>> &rValues,
+    const ProcessInfo &rCurrentProcessInfo)
+{
+    if (rVariable == EMBEDDED_VELOCITY) {
+        const auto &r_geom = this->GetGeometry();
+        const auto &r_integration_points = r_geom.IntegrationPoints(this->GetIntegrationMethod());
+        const std::size_t n_gauss_pts = r_integration_points.size();
+        rValues.resize(n_gauss_pts);
+        for (std::size_t i_gauss = 0; i_gauss < n_gauss_pts; ++i_gauss) {
+            rValues[i_gauss] = this->GetValue(EMBEDDED_VELOCITY);
+        }
+    } else {
+        TBaseElement::GetValueOnIntegrationPoints(rVariable, rValues, rCurrentProcessInfo);
+    }
+}
+
+template <class TBaseElement>
+void EmbeddedFluidElement<TBaseElement>::GetValueOnIntegrationPoints(
+    const Variable<double> &rVariable,
+    std::vector<double> &rValues,
+    const ProcessInfo &rCurrentProcessInfo)
+{
+    TBaseElement::GetValueOnIntegrationPoints(rVariable, rValues, rCurrentProcessInfo);
+}
```

Comment:
I think if you use the `using xxx` for those functions you don't have to implement empty functions => see #3412